common/djangoapps/third_party_auth/middleware.py

```diff
@@ -45,7 +45,7 @@ def process_view(self, request, view_func, view_args, view_kwargs):  # pylint: d
         collection of user consent for sharing data with a linked third-party
         authentication provider.
         """
-        running_pipeline = request.session.get('partial_pipeline')
+        running_pipeline = request.session.get('partial_pipeline_token')
```

Comment:
Sigh. Since we're DB-backing now, this is more intensive than just getting the value off the session, so please switch this over to using `pipeline.get()`.